src/components/eyewitness/eyewitness.tsx

```diff
@@ -1,96 +1,136 @@
-import { Button, LoadingOverlay, Stack, TextInput, Title } from "@mantine/core";
+import { Button, Stack, TextInput, Title } from "@mantine/core";
 import { useForm } from "@mantine/form";
-import { useCallback, useState } from "react";
+import { useCallback, useState, useEffect } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
-import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import { RenderComponent } from "../UserGuide/UserGuide";
 
-const title = "EyeWitness";
-const description_userguide =
-    "EyeWitness takes screenshots of websites, provides information about the server header, and identifies default credentials (if known). It presents this information in a HTML report. " +
-    " \n\nEyeWitness's information page:https://www.kali.org/tools/eyewitness/#eyewitness" +
-    " \n\nHow to use EyeWitness:" +
-    "\n\nStep 1: Create a plain text file on your local drive and add URLs to it. Each URL must be on its own line. Add the file path to the text file in the first field." +
-    "\n\nStep 2: Add the file path for where you want the output saved in the second field." +
-    "\n\nStep3: Add a number in the third field for the maximum number of seconds for EyeWitness to try and screenshot a webpage, e.g. 20. " +
-    "\n\nStep 4: Press the scan button. ";
-
+/**
+ * Represents the form values for the EyeWitness component.
+ */
 interface FormValues {
```

Comment:
Should be named FormValuesType.
```suggestion
interface FormValuesType {
```